docs/sources/mimir/configure/configure-kafka-backend.md

```diff
@@ -12,6 +12,30 @@ weight: 130
 Grafana Mimir supports using Kafka for the first layer of ingestion. This is an experimental feature released in Mimir 2.14.
 This page is incomplete. It will be updated as the ingest storage feature matures and moves out of the experimental phase.
 
+## Configuring ingest storage
+
+The following configuration flags enable Grafana Mimir to use the ingest storage through Kafka backend:
+
+- `-ingest-storage.enabled=true`<br />
+  The ingest storage architecture must be explicitly enabled in all Mimir components.
+
+- `-ingest-storage.kafka.address=<host:port>`<br />
+  The `<host:port>` is the address of Kafka broker to bootstrap the connection.
+
+- `-ingest-storage.kafka.topic=<name>`<br />
+  The `<name>` is the name of the Kafka topic, that will be used for ingesting data.
+- `-ingest-storage.kafka.auto-create-topic-default-partitions=<number>`<br />
+  If the configured topic doesn't not exists in Kafka backend, the Mimir components (either consumers or producers),
+  will create the topic on first access. The `<number>` parameter sets the number of partitions to create when the topic is automatically created. The number of partitions must be at least the number of ingesters in one zone.
+
+Additionally, there configuration options are recommended, when Grafana Mimir runs in the ingest storage architecture:
```

Comment:
```suggestion
Additionally, you can use these recommended configuration options when running Grafana Mimir with ingest storage architecture:
```

Reply:
Are there additional options to list here? If not, we should append this option to the above list and note that it is optional but recommended.